lib/changes.go

```diff
@@ -0,0 +1,49 @@
+package lib
+
+import (
+	"context"
+
+	"github.com/qri-io/qri/changes"
+	"github.com/qri-io/qri/dsref"
+)
+
+// ChangeReportParams defines parameters for diffing two sources
+type ChangeReportParams struct {
+	LeftRefstr  string
+	RightRefstr string
+}
```

Comment:
```suggestion
type ChangeReportParams struct {
	LeftRefstr  string `json:"left"`
	RightRefstr string `json:"right"`
}
```

adding this would be really helpful for desktop!

Reply:
Totally, this was just a rough mashup of params to get things going